package.json

```diff
@@ -2,26 +2,15 @@
   "name": "cozy-desktop",
   "version": "0.7.0",
   "description": "File Synchronization Client for Cozy Cloud",
-  "main": "app.html",
-  "window": {
-    "position": "center",
-    "width": 600,
-    "height": 600,
-    "show": true,
-    "resizable": false,
-    "toolbar": false,
-    "icon": "client/public/icon/main_icon.png",
-    "title": "Cozy Desktop"
-  },
-  "scripts": {
-    "lint": "gulp lint",
-    "test": "gulp test",
-    "build": "gulp build",
-    "prepublish": "gulp build",
-    "start": "nw"
+  "homepage": "https://github.com/cozy-labs/cozy-desktop",
+  "author": "Cozy Cloud <contact@cozycloud.cc> (https://cozycloud.cc/)",
+  "license": "LGPL-3.0",
+  "bugs": {
+    "url": "https://github.com/cozy-labs/cozy-desktop/issues"
   },
+  "main": "./lib/bin/cli.js",
```

Comment:
Well, https://docs.npmjs.com/files/package.json#main explains that this field is used when someone `require` the cozy-desktop module. So, I think it should be neither `bin/cozy-desktop` or `lib/bin/cli.js`, but something useful for building on top of cozy-desktop: `lib/app.js` (for example, if someone wants to build an alternative GUI, it will this file that he will want to require).